db/migrate/20181201021612_create_effects.rb

```diff
@@ -0,0 +1,13 @@
+Sequel.migration do
+  change do
```

Comment:
Migrations will usually include a 'change' block, or an 'up' block and a 'down' block. 'change' blocks only allow commands that Sequel knows how to reverse, and so it will create the 'up' and 'down' for you. Simple migrations usually involve only 'change', usually you'll see 'up' and 'down' for data migrations. Like, if you need to move/copy/reformat the current values in some column.